core/server/api/canary/members.js

```diff
@@ -604,7 +604,10 @@ module.exports = {
                     invalid,
                     createdBy
                 });
-            }).then(() => {
+            }).then((result) => {
+                invalid.errors = invalid.errors.concat(result.invalid.errors);
```

Comment:
Yesss! +100 points for getting rid of those 2 extra parameters!

Reply:
:hugs: